alma/views/templates/hook/order_refund_bs4.tpl

```diff
@@ -107,7 +107,7 @@
                 </div>
             </div>
             <div class="text-right">
-            <button type="submit" class="button btn btn-primary button-medium pull-right">
+            <button type="submit" class="button btn btn-primary">
```

Comment:
with just a `<div class="clearfix"></div>` after the <button> don't fix this issue ? For don't add class in the button.

Reply:
It works like that